api/src/test/java/org/openmrs/notification/MessageServiceTest.java

```diff
@@ -113,4 +119,68 @@ public void sendMessage_shouldSendMessage() throws MessageException {
 		}
 	}
 	
+	@Test
+	public void setMessagePreparator_shouldSetMessagePreparator() throws MessageException {
+		MessagePreparator messagePreparator = new VelocityMessagePreparator();
+		MessageService messageService = Context.getMessageService();
+		messageService.setMessagePreparator(messagePreparator);
+		assertEquals(messagePreparator, messageService.getMessagePreparator());
+	}
+
+	@Test
+	public void setMessageSender_shouldSetMessageSender() throws MessageException {
+		MessageSender messageSender = new MailMessageSender();
+		MessageService messageService = Context.getMessageService();
```

Comment:
Can you mock this? https://wiki.openmrs.org/display/docs/Mock+Doc . It is a little overkill to load the whole context for a test.